modules/nf-core/fastqscreen/buildfromindex/main.nf

```diff
@@ -0,0 +1,67 @@
+process FASTQSCREEN_BUILDFROMINDEX {
+    label 'process_single'
+
+    conda "fastq-screen"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/fastq-screen%3A0.15.3--pl5321hdfd78af_0':
+        'quay.io/biocontainers/fastq-screen:0.15.3--pl5321hdfd78af_0'}"
+
+    input:
+    val(id)
```

Comment:
Can we call this something else? It is all the names collected together, so genome_names? At least ids rather than id